dist/package.sh

```diff
@@ -83,11 +83,11 @@ for target in "${targets[@]}"; do
     else
         install -D "node_modules/$target" "$DIR/node_modules/$target"
     fi
-    sed -i "/sourceMappingURL/d" "$DIR/node_modules/$target"
+    sed -i.bak "/sourceMappingURL/d" "$DIR/node_modules/$target"
 done
 
 echo "+ Update version number..."
-# Note: The `-i.bak` notation is requires so that the sed command works both on Linux
+# Note: The `-i.bak` notation is required so that the sed command works both on Linux
 # and on macOS: https://stackoverflow.com/q/5694228/284318
```

Comment:
And since we use `sed` multiple times, I'd like this comment updated and moved to the top one (or just removed altogether).

Reply:
I removed the comment.